src/service_aggregator.py

```diff
@@ -165,19 +174,19 @@ async def entry(message, guid, coalesce_type, caller) -> (dict, int):
     # We told the world what we can do!
     # Workflow will be a list of the functions, and the parameters if there are any
 
+    read_from_cache = bypass_cache or overwrite_cache
```

Comment:
The name of this variable is the reverse of what it does?

Reply:
Oh shoot, thanks